### ISSUE TITLE
logging.properties causes large number of logs in the log file

### ISSUE THREAD

**Issue by GoogleCodeExporter** [jeewan2015/grails-activiti-plugin#18]
```
What steps will reproduce the problem?
Run grails prod tomcat deploy

What is the expected output? What do you see instead?
Reasonable logs, but because of the catch all .level = FINE, the unecessary 
logs are recorded.
Sample of the logs are:
Jul 26, 2011 8:03:09 AM org.apache.tomcat.util.digester.Digester startElement
FINE: startElement(,,tag-class)
Jul 26, 2011 8:03:09 AM org.apache.tomcat.util.digester.Digester startElement
FINE:   Pushing body text ''
Jul 26, 2011 8:03:09 AM org.apache.tomcat.util.digester.Digester startElement
FINE:   New match='taglib/tag/tag-class'
Jul 26, 2011 8:03:09 AM org.apache.tomcat.util.digester.Digester startElement
FINE:   No rules found matching 'taglib/tag/tag-class'.
Jul 26, 2011 8:03:09 AM org.apache.tomcat.util.digester.Digester characters
FINE: characters(com.opensymphony.module.sitemesh.taglib.page.ParamTag)
Jul 26, 2011 8:03:09 AM org.apache.tomcat.util.digester.Digester endElement
FINE: endElement(,,tag-class)
Jul 26, 2011 8:03:09 AM org.apache.tomcat.util.digester.Digester endElement
FINE:   match='taglib/tag/tag-class'
Jul 26, 2011 8:03:09 AM org.apache.tomcat.util.digester.Digester endElement
FINE:   bodyText='com.opensymphony.module.sitemesh.taglib.page.ParamTag'
Jul 26, 2011 8:03:09 AM org.apache.tomcat.util.digester.Digester endElement
FINE:   No rules found matching 'taglib/tag/tag-class'.
Jul 26, 2011 8:03:09 AM org.apache.tomcat.util.digester.Digester endElement
FINE:   Popping body text ''
Jul 26, 2011 8:03:09 AM org.apache.tomcat.util.digester.Digester 
ignorableWhitespace
FINE: ignorableWhitespace(

What version of the product are you using? On what operating system?
activiti plugins 5.6

Please provide any additional information below.


```

Original issue reported on code.google.com by `tmcchan...@gmail.com` on 26 Jul 2011 at 12:57

**Comment by GoogleCodeExporter** [comment]
```
logging.properties is come with the Activiti distribution, I simply 
incorporated it into the plugin. Did you have any suggestion to improve it? 


```

Original comment by `limchee...@vobject.com` on 26 Jul 2011 at 2:21

**Comment by GoogleCodeExporter** [comment]
```
What I did is to delete the file altogether and manage the logging separately
```

Original comment by `tmcchan...@gmail.com` on 26 Jul 2011 at 2:41

**Comment by GoogleCodeExporter** [comment]
```
Thanks for input, this is what I think also, I will remove the 
logging.properties file in next release of the plugin
```

Original comment by `limchee...@vobject.com` on 26 Jul 2011 at 5:32

**Comment by GoogleCodeExporter** [comment]
Original comment by `limchee...@vobject.com` on 26 Jul 2011 at 5:37
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
Fixed on 5.7 released.
```

Original comment by `limchee...@vobject.com` on 19 Aug 2011 at 10:44
- Changed state: **Fixed**
